README.md

```diff
@@ -69,52 +69,52 @@ Supported protocols:
 - `ftp` Plain FTP
 - `ftps` Implicit FTP over TLS
 
-_Note:_ The hostname must be the external IP address to accept external connections. `0.0.0.0` will listen on any available hosts for server and passive connections.  
+_Note:_ The hostname must be the external IP address to accept external connections. `0.0.0.0` will listen on any available hosts for server and passive connections.
```

Comment:
These spaces are actually required for markdown to render the next line _on_ the next line.

With trailing spaces:
![image](https://user-images.githubusercontent.com/15315657/53188468-a8986d00-35c2-11e9-8e43-d3348ec538fd.png)

Without trailing spaces:
![image](https://user-images.githubusercontent.com/15315657/53188437-95859d00-35c2-11e9-9c93-2fb6008da67f.png)

If you're using VSCode you can change this in `settings.json`:

```
    "files.trimTrailingWhitespace": true,
    "[markdown]": {
        "files.trimTrailingWhitespace": false
    },
```

Reply:
It'd be nice to work on a nicer, simpler readme and better API documentation.
#dreams